README.md

```diff
@@ -20,38 +20,40 @@ with large data-models.
 - **Extendable** - simple to extend and customize using Go templates.
 
 ## Quick Installation
+
 ```console
 go install entgo.io/ent/cmd/ent@latest
 ```
 
 For proper installation using [Go modules], visit [entgo.io website][entgo install].
 
 ## Docs and Support
-The documentation for developing and using ent is available at: https://entgo.io
+
+The documentation for developing and using ent is available at: [https://entgo.io]([https://entgo.io).
 
 For discussion and support, [open an issue](https://github.com/ent/ent/issues/new/choose) or join our [channel](https://gophers.slack.com/archives/C01FMSQDT53) in the gophers Slack.
 
 ## Join the ent Community
-Building `ent` would not have been possible without the collective work of our entire community. We maintain a [contributors page](doc/md/contributors.md)
-which lists the contributors to this `ent`. 
+
+Building `ent` would not have been possible without the collective work of our entire community. We maintain a [contributors page](doc/md/contributors.md).
```

Comment:
The extra dot in not correct. Markdown single line breaks are not translated to HTML line breaks.

Reply:
OK. I will fix it.